pkg/utils/stringreplacer.go

```diff
@@ -0,0 +1,18 @@
+package utils
```

Comment:
I dont get the point of this type at all, we can talk about it but imo it should be removed.

Reply:
Yeah lets talk - you created this for the downloader package to allow string templating but it looked very generic so I moved it to utils in case we wanted to use it elsewhere....